change/@fluentui-react-dialog-c4b3b805-3dcc-4caa-9c9e-ceb9a1580a1a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29391&commitId=d09808ce3afc302d62b452a5ffdb04a5ea51a3b3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Dialog</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.actions position start - High Contrast.chromium.png|31997| Changed|
|Dialog.actions position start & position end.chromium.png|21053| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|22941| Changed|
|Dialog.alert - RTL.chromium.png|20521| Changed|
|Dialog.default - Dark Mode.chromium.png|22855| Changed|
|Dialog.alert - High Contrast.chromium.png|31774| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|32625| Changed|
|Dialog.Fluid Actions End.chromium.png|22012| Changed|
|Dialog.actions position start.chromium.png|20718| Changed|
|Dialog.actions position start - RTL.chromium.png|20663| Changed|
|Dialog.default - High Contrast.chromium.png|31774| Changed|
|Dialog.alert - Dark Mode.chromium.png|22855| Changed|
|Dialog.Fluid Actions Start.chromium.png|22319| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|21031| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|23358| Changed|
|Dialog.alert.chromium.png|20550| Changed|
|Dialog.default - RTL.chromium.png|20521| Changed|
|Dialog.nested - RTL.chromium.png|16221| Changed|
|Dialog.nested - High Contrast.chromium.png|25977| Changed|
|Dialog.nested - Dark Mode.chromium.png|19399| Changed|
|Dialog.default.chromium.png|20550| Changed|
|Dialog.no actions - Dark Mode.chromium.png|18852| Changed|
|Dialog.no title & no actions - RTL.chromium.png|13173| Changed|
|Dialog.no actions - High Contrast.chromium.png|24380| Changed|
|Dialog.nested.chromium.png|16183| Changed|
|Dialog.no actions.chromium.png|15863| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|20579| Changed|
|Dialog.no actions - RTL.chromium.png|15839| Changed|
|Dialog.no title & no actions.chromium.png|13224| Changed|
|Dialog.no title - Dark Mode.chromium.png|20164| Changed|
|Dialog.no title - High Contrast.chromium.png|27868| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|16001| Changed|
|Dialog.no title.chromium.png|17910| Changed|
|Dialog.no title - RTL.chromium.png|17913| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|38153| Changed|
|Dialog.non-modal - High Contrast.chromium.png|45453| Changed|
|Dialog.non-modal - RTL.chromium.png|10657| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|167201| Changed|
|Dialog.scroll long content - RTL.chromium.png|132838| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|19730| Changed|
|Dialog.non-modal.chromium.png|10677| Changed|
|Dialog.title custom action - RTL.chromium.png|16726| Changed|
|Dialog.title custom action.chromium.png|16729| Changed|
|Dialog.title custom action - High Contrast.chromium.png|25314| Changed|
|Dialog.scroll long content.chromium.png|132917| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|131534| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>